tests/test_workers/test_pkg_managers/test_general_js.py

```diff
@@ -114,7 +114,8 @@ def test_download_dependencies(
 
     dep1_source_path = os.path.join(download_dir, "angular-devkit-architect-0.803.26.tgz")
     dep1_dest_path = os.path.join(
-        download_dir, "@angular-devkit/architect/angular-devkit-architect-0.803.26.tgz",
+        download_dir,
+        "@angular-devkit/architect/angular-devkit-architect-0.803.26.tgz",
```

Comment:
Well now we have one formatting change in this commit and the rest in the previous one :smile: 

If `black` asked for this change it's OK to include it in the same commit, I just didn't understand why black did that (since the code didn't change)

Reply:
Yes, it was an issue with `black`. There was a little miscommunication on my part, sorry :D